hoomd/hpmc/integrate.py

```diff
@@ -330,19 +336,18 @@ class Sphere(_HPMCIntegrator):
 
```

Comment:
Here is where I would include the documentation about `fugacity`, `interaction_matrix`, `a`, and `d`. Also I would change doc format to be something like
````
Sphere parameters

shape (particle type, dict): defines the shape of the object.
    Keys:
        * *diameter* (float, **required) - diameter of the sphere (distance units)
        * ...

d (particle type, float): the size of displacement trial moves

interaction_matrix ((particle type, particle type), bool): Whether to include overlaps between type 1 and type 2

...
````
Something similar would apply to each of the integrators. If we merge before doing this, we need to raise an issue since without this documentation, it will be hard to learn to use the integrators.